docs/index.md

```diff
@@ -0,0 +1,191 @@
+---
+hide:
+  - navigation
+---
+
+![Merlin Banner](./assets/merlin_banner.png){ width="75%" }
+
+----
+
+# Merlin
+
+Empower your projects with Merlin, a cloud-based workflow manager designed to facilitate scalable and reproducible workflows, particularly suited for running many simulations and iterative procedures.
+
+[On GitHub :fontawesome-brands-github:](https://github.com/LLNL/merlin){: .md-button .md-button--primary }
+
+----
+
+## Why Merlin?
+
+Workflows, applications and machines are becoming more complex, but subject matter experts need to devote time and attention to their applications and often require fine command-line level control. Furthermore, they rarely have the time to devote to learning workflow systems.
+
+With the expansion of data-driven computing, the HPC scientist needs to be able to run more simulations through complex multi-component workflows.
+
+**Merlin targets HPC workflows that require many simulations**.
+
+## Getting Started
+
+Merlin can be installed via pip in your own virtual environment.
+
+1. First, create a virtual environment:
+
+    ```bash
+    python -m venv merlin_venv
+    ```
+
+2. Now activate the virtual environment:
+
+    ```bash
+    source merlin_venv/bin/activate
+    ```
+
+3. Finally, install Merlin with pip:
+
+    ```bash
+    pip install merlin
+    ```
+
+Next, let's create a folder to store examples and move into it:
+
+```bash
+mkdir merlin_examples ; cd merlin_examples/
+```
+
+We can now download Merlin's built-in "Hello, World!" example:
+
+```bash
+merlin example hello
+```
+
+Before we run this example, let's set up a containerized server that Merlin can connect to.
+
+1. First, initialize the server files:
+
+    ```bash
+    merlin server init
+    ```
+
+2. Next, start the server:
+
+    ```bash
+    merlin server start
+    ```
+
+3. Finally, copy the `app.yaml` configuration file from `merlin_server/` to your current directory:
+
+    ```bash
+    cp merlin_server/app.yaml .
+    ```
+
```

Comment:
Maybe include a `merlin info` to check servers?

Reply:
yeah that's a good idea